CMakeLists.txt

```diff
@@ -18,34 +30,48 @@ include(${_project_options_SOURCE_DIR}/Index.cmake)
 # run_vcpkg()
 
 # Set the project name and language
-project(myproject LANGUAGES CXX)
+project(myproject LANGUAGES CXX C)
+
+if(GENERATOR_IS_MULTI_CONFIG AND NOT CMAKE_BUILD_TYPE)
+  # Make sure that all supported configuration types have their
+  # associated conan packages available. You can reduce this
+  # list to only the configuration types you use, but only if one
+  # is not forced-set on the command line for VS
+  set(CMAKE_CONFIGURATION_TYPES
+      Debug
+      Release
+      RelWithDebInfo
+      MinSizeRel)
+endif()
```

Comment:
Why don't we add this to project_options? 
https://github.com/cpp-best-practices/project_options/blob/main/src/StandardProjectSettings.cmake

Reply:
I wasn't sure where it was best to live. I'm also not 100% sure yet if it's working as intended 😄 should probably be double checked. It does seem on the CI that we are building all BUILD_TYPEs for Conan even when we don't need to.